### DISCUSSION TITLE
In math.cases the branches are small

### DISCUSSION THREAD

**Discussion by egri-nagy** [typst/typst#4005] [General]
### Description

When using math.cases the branches get small by default. Expected behavior: no change in size.
Current workaround: using `inline`.
Is there a parameter to set the default size?

### Reproduction URL

https://typst.app/project/rT2xNGC4wOIL1YDFWEkn7i

### Operating system

Linux, macOS

### Typst version

- [X] I am using the latest version of Typst

**Comment by laurmaedje** [comment]
`inline` is your best bet at the moment, there is no other way
